clevertap.js

```
@@ -5856,12 +5901,119 @@
+      return {
+        serviceWorkerPath,
+        skipDialog,
+        okCallback,
+        subscriptionCallback,
+        rejectCallback
+      };
+    }
+
+    return {
+      serviceWorkerPath: undefined,
+      skipDialog: (_displayArgs$ = displayArgs[5]) !== null && _displayArgs$ !== void 0 ? _displayArgs$ : undefined,
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)